test/e2e/framework/auth/.import-restrictions

```diff
@@ -1,9 +1,12 @@
 # This E2E framework sub-package is currently allowed to use arbitrary
-# dependencies, therefore we need to override the restrictions from
-# the parent .import-restrictions file.
+# dependencies except of k/k/pkg, therefore we need to override the
+# restrictions from the parent .import-restrictions file.
 #
 # At some point it may become useful to also check this package's
 # dependencies more careful.
 rules:
+  - selectorRegexp: "^k8s[.]io/kubernetes/pkg"
+    forbiddenPrefixes: [ "" ]
```

Comment:
Instead of `forbiddenPrefixes: [ "" ]`, can you use `allowedPrefixes: []`? I find that clearer because it matches how one would describe this rule in plain English: "from k8s.io/kubernetes/pkg, allow nothing".

Reply:
done

Reply:
However, for me "forbid everything" sounds closer to what we want to do because it sounds stricter.